qiskit/pulse/channels/output_channel.py

```diff
@@ -9,26 +9,75 @@
 Channels support signal output.
 """
 from abc import abstractmethod
+from typing import Tuple
 
+from qiskit.pulse.exceptions import PulseError
 from .pulse_channel import Channel
 
 
+class LoRange:
+    """Range of LO frequency."""
+
+    def __init__(self, lower_bound: float, upper_bound: float):
+        self._lb = lower_bound
+        self._ub = upper_bound
+
+    def includes(self, lo_freq: float) -> bool:
+        """Return if this range includes `lo_freq` or not.
+
+        Args:
+            lo_freq: LO frequency to be checked
+
+        Returns:
+            bool: True if lo_freq is included in this range, otherwise False
+        """
+        if self._lb <= lo_freq <= self._ub:
+            return True
+        return False
+
+    @property
+    def lower_bound(self) -> float:
+        """Lower bound of this LO range"""
+        return self._lb
+
+    @property
+    def upper_bound(self) -> float:
+        """Upper bound of this LO range"""
+        return self._ub
+
+    def __repr__(self):
+        return "%s(%f, %f)" % (self.__class__.__name__, self._lb, self._ub)
+
+
 class OutputChannel(Channel):
     """Output Channel."""
 
     @abstractmethod
-    def __init__(self, index: int = None, lo_frequency: float = None):
+    def __init__(self,
+                 index: int,
+                 lo_frequency: float = None,
```

Comment:
Could this be `lo_freq`?

Reply:
I added the change to [#44](https://github.com/nkanazawa1989/qiskit-terra/pull/44) as lo related modification.